Prac07/guitar.py

```diff
@@ -0,0 +1,18 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}) : ${:.2f} added.".format(self.name, self.year, self.cost)
+
```

Comment:
Re-usability continued:

guitar.py
"""    def guitar_details(self):
           return "{:>20} ({}), worth ${:10,.2f}".format(self.name, self.year, self.cost)"""

I'll leave the rest to you on this part.